### PR TITLE
[FEAT] added single flag to set all the data/keys/config to one containing dir

### PR DESCRIPTION
**Pull request by aricart**
added `--all-dirs` option to store config, data, keys under the same directory - this is useful for generating test configurations

### CODE REVIEW

File: cmd/env_test.go [hunk 1]
```diff
@@ -71,3 +72,17 @@ func TestEnv_FailsBadAccount(t *testing.T) {
 	require.Error(t, err)
 	require.Contains(t, err.Error(), "\"A\" not in accounts for operator \"O\"")
 }
+
+func TestAllDir(t *testing.T) {
+	p := MakeTempDir(t)
+	defer func() {
+		os.RemoveAll(p)
+	}()
```
Comment (piotrpio) [inline cmd/env_test.go:80]:
```suggestion
	defer os.RemoveAll(p)
```

File: cmd/root.go [hunk 2]
```diff
@@ -33,6 +33,7 @@ import (
 var ConfigDirFlag string
 var KeysDirFlag string
 var DataDirFlag string
+var AllDirFlag string
```
Comment (piotrpio) [inline cmd/root.go:36]: I would suggest keeping all config flags under one `var` block for clarity:
```
var (
  ConfigDirFlag string
  KeysDirFlag string
  DataDirFlag string
  AllDirFlag string
  ...
)
```

File: cmd/root.go [hunk 3]
```diff
@@ -256,6 +263,6 @@ func HoistRootFlags(cmd *cobra.Command) *cobra.Command {
 	cmd.PersistentFlags().StringVarP(&ConfigDirFlag, "config-dir", "", "", "nsc config directory")
 	cmd.PersistentFlags().StringVarP(&DataDirFlag, "data-dir", "", "", "nsc data store directory")
 	cmd.PersistentFlags().StringVarP(&KeysDirFlag, "keystore-dir", "", "", "nsc keystore directory")
-
+	cmd.PersistentFlags().StringVarP(&AllDirFlag, "all-dirs", "H", "", "sets --config-dir, --data-dir, and --keystore-dir to the same value")
```
Comment (piotrpio) [inline cmd/root.go:266]: Why `-H` as the shorthand?

Comment (aricart) [reply cmd/root.go:266]: "home" - but that means something so I didn't want to make the human flag that

**Review by piotrpio — APPROVED** [review]
LGTM

**Pull request merged**